packages/dds/tree/src/events/interop.ts

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IEvent } from "@fluidframework/core-interfaces";
+import type { Listeners } from "./listeners.js";
+
+// TODO: this file is currently unused. Use it or remove it.
```

Comment:
For this PR? Or for the future?

Reply:
Future. I assume this code has a reason it was written, but it does not document any, and isn't used, so I put it here and noted that.

I believe this is the logic to adapt tree's events to work with the framework/node event APIs.